processor/processor.go

```diff
@@ -570,3 +575,35 @@ func (p *processor) parseMarkers() {
 		}
 	}
 }
+
+func extractConstantsForAliasedString(pkg *loader.Package, name string) []types.Const {
+	consts := []types.Const{}
+	for _, f := range pkg.Syntax {
+		for _, d := range f.Decls {
+			switch n := d.(type) {
+			case *ast.GenDecl:
+				if n.Tok == token.CONST {
+					for _, s := range n.Specs {
+						switch v := s.(type) {
+						case *ast.ValueSpec:
+							if id, ok := v.Type.(*ast.Ident); ok && id.String() == name {
+								if len(v.Values) == 1 {
+									if b, ok := v.Values[0].(*ast.BasicLit); ok {
+										consts = append(consts, types.Const{
+											// Remove the '"' signs from the start and end of the value
+											Name: b.Value[1 : len(b.Value)-1],
+											Doc:  v.Doc.Text(),
+										})
+									}
+								}
+							}
+						default:
+						}
+					}
+				}
+			default:
+			}
+		}
+	}
+	return consts
+}
```

Comment:
I think we could try having less level of indentation to increase the readability a bit since it's not easy to read as is. Feel free to add or change the comments.
```suggestion
func lookupConstantValuesForAliasedType(pkg *loader.Package, aliasTypeName string) []types.ConstantValue {
	values := []types.ConstantValue{}
	for _, file := range pkg.Syntax {
		for _, decl := range file.Decls {
			node, ok := decl.(*ast.GenDecl)
			if !ok || node.Tok != token.CONST {
				continue
			}
			for _, spec := range node.Specs {
				// look for constant declaration
				v, ok := spec.(*ast.ValueSpec)
				if !ok {
					continue
				}
				// value type must match the alias type name and have exactly one value
				if id, ok := v.Type.(*ast.Ident); !ok || id.String() != aliasTypeName || len(v.Values) != 1 {
					continue
				}
				// convert to a basic type to access to the value
				b, ok := v.Values[0].(*ast.BasicLit)
				if !ok {
					continue
				}
				values = append(values, types.ConstantValue{
					// remove the '"' signs from the start and end of the value
					Name: b.Value[1 : len(b.Value)-1],
					Doc:  v.Doc.Text(),
				})
			}
		}
	}
	return values
}
```